app-typescript/components/CheckButtonGroup.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+interface IProps {
+    orientation?: 'horizontal' | 'vertical'; // defaults to 'horizontal'
+    grid?: boolean;
+    align?: 'left' | 'right' | 'center'; // defaults to 'left'
+    children: React.ReactNode;
```

Comment:
You don't need to put children in the interface unless you want to limit it. For example, if the component doesn't support `children` like a Button, you can specify `children?: never;` so the code doesn't compile if it's passed. Or you can make children accept a function like `children: () => number;`. If any React component works as children, you don't need to specify it in the interface.